pkgs/by-name/mu/muvm/package.nix

```diff
@@ -0,0 +1,109 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  libkrun,
+  passt,
+  dhcpcd,
+  systemd,
+  udev,
+  pkg-config,
+  procps,
+  fex,
+  writeShellApplication,
+  coreutils,
+  makeBinaryWrapper,
+# TODO: Enable again when sommelier is not broken.
+# For now, don't give false impression of sommelier being supported.
+# sommelier,
+# withSommelier ? false,
+}:
+let
+  # TODO: Setup setuid wrappers.
+  # E.g. FEX needs fusermount for rootfs functionality
+  initScript = writeShellApplication {
+    name = "muvm-init";
+    runtimeInputs = [
+      coreutils
+    ];
+    text = ''
+      if [[ ! -f /etc/NIXOS ]]; then exit; fi
+
+      ln -s /run/muvm-host/run/current-system /run/current-system
+      ln -s /run/muvm-host/run/opengl-driver /run/opengl-driver
```

Comment:
Not every system has `/run/opengl-driver` even if it is NixOS

Reply:
I think it's better if they don't get created because I think reading a broken symlink will result in an error which isn't "no entry" and that could cause issues for certain programs which checks for the existence of the directory.

Reply:
Probably a sign of buggy software, but fair enough, I made it conditional